app/konnect/dev-portal/applications/application-overview.md

```diff
@@ -1,72 +1,30 @@
-support application registration for the Service and are managed by
-{{site.konnect_saas}}.
+In [cloud.konghq.com](https://cloud.konghq.com), admins can access a list of the installed authentication plugins via the **Runtime Manager**. See [Enable Application Registration for a Service](/konnect/dev-portal/applications/enable-app-reg/) for more information about authentication flows.
 
-To disable or delete a plugin that was enabled by app registration,
-you must disable app registration itself. You can't use the toggle in the
-Plugins pane on a Service version, as the toggle is unavailable for
-{{site.konnect_short_name}}-managed plugins.
+Once a developer is [granted access](/konnect/dev-portal/access-and-approval/manage-devs/) to the {{site.konnect_short_name}} Dev Portal, they will be able to create, edit, and delete applications. These modifications are all managed on the **My Apps** page. **My Apps** allows you to view all of the registered applications. Clicking on individual applications from this page opens a detailed overview of an application. 
 
-![Konnect Enable App Registration with OIDC](/assets/images/docs/konnect/konnect-enable-app-reg-oidc-toggle.png)
+You can do the following through the application details page:
 
-If using a [declarative configuration](/konnect/configure/runtime-manager/runtime-groups/declarative-config)
-file to manage your Service, these plugins appear in the file. **Do not**
-delete or edit them through declarative configuration, as it will break your Service.
+- [Edit](#edit-an-application) the name, reference ID, and description of an application.
+- [Generate or delete credentials](/konnect/dev-portal/applications/dev-gen-creds/).
+- View a catalog of Services that can be [registered with the application](/konnect/dev-portal/applications/dev-reg-app-service), if no Services are registered yet.
+- View the status of an application registration to a Service.
 
-To help differentiate the application registration plugins,
-{{site.konnect_short_name}} automatically adds two metadata tags:
-`konnect-managed-plugin` and `konnect-app-registration`.
+The guides within this section cover the following functionality: 
 
-For example, if you enable application registration from the
-{{site.konnect_short_name}} GUI and run `deck dump`, you should see
-an entry like this for the ACL plugin:
+* [How to create, edit and delete applications](/konnect/dev-portal/applications/dev-apps)
+* [How to enable and disable app registration](/konnect/dev-portal/applications/enable-app-reg)
+* [How to manage registration requests](/konnect/dev-portal/applications/manage-app-reg-requests)
+* [How to manage app Service connections](/konnect/dev-portal/applications/manage-app-connections)
+* [How to register or unregister an application for a Service](/konnect/dev-portal/applications/dev-reg-app-service)
+* [How to generate credentials for an application](/konnect/dev-portal/applications/dev-gen-creds/)
 
-```yaml
```

Comment:
Are you planning on moving this section (old lines 46-72) elsewhere after this PR is merged? I don't want to lose this info.